code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -93,6 +99,102 @@
 		spawn(200)
 			M.current.verbs += /client/proc/vampire_rejuvinate
 
+/client/proc/vampire_returntolife()
```

Comment:
Yikes. This proc should call something in medical code to do the resurrection, half this code isn't vampire-specific at all